VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardActivity.java

```diff
@@ -0,0 +1,89 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.MainApplication;
+import org.edx.mobile.logger.Logger;
+import org.edx.mobile.util.images.ImageCacheManager;
+import org.edx.mobile.view.custom.CourseImageHeader;
+
+
+/**
+ * TODO - it is just a place holder for now. as we need to use it
+ * to navigation to new views.
+ */
+public class CourseDashboardActivity extends CourseBaseActivity {
```

Comment:
Is now the time to start building this stuff around fragments instead of activities?